backend/eventd/eventd.go

```diff
@@ -185,11 +185,6 @@ func (e *Eventd) handleMessage(msg interface{}) error {
 		return err
 	}
 
-	err = e.store.UpdateEvent(ctx, event)
```

Comment:
I think you actually need to remove the other one, since the if condition returns if true.